tests/mocks/mock_config.py

```diff
@@ -29,3 +29,4 @@ def _fill_patchers(self):
         patchers.append(
             patch("builtins.open", mock_open(read_data=self.yaml_file_content)))
         patchers.append(patch("sys.argv", self.args))
+        patchers.append(patch('numba.cuda.is_available', MagicMock(True)))
```

Comment:
without this, all of our test default configs would use KIND_CPU because cuda isn't available.